policy-report/kube-bench-adapter/pkg/kubebench/kubebench.go

```diff
@@ -0,0 +1,177 @@
+package kubebench
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"strings"
+	"time"
+
+	kubebench "github.com/aquasecurity/kube-bench/check"
+	batchv1 "k8s.io/api/batch/v1"
+	apiv1 "k8s.io/api/core/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	yaml "k8s.io/apimachinery/pkg/util/yaml"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/klog"
+)
+
+func getClientSet(kubeconfigPath string) (*kubernetes.Clientset, error) {
+	var kubeconfig *rest.Config
+
+	cfg, err := rest.InClusterConfig()
+	if err != nil {
+		cfg, err = clientcmd.BuildConfigFromFlags("", kubeconfigPath)
+		if err != nil {
+			klog.Fatalf("Error building kubeconfig: %s", err.Error())
+			return nil, err
+		}
+	}
+	kubeconfig = cfg
+
+	clientset, err := kubernetes.NewForConfig(kubeconfig)
+	if err != nil {
+		return nil, err
+	}
+
+	return clientset, nil
+
+}
+func RunJob(kubeconfig string, kubebenchYAML, kubebenchImg, kubebenchVersion, kubebenchBenchmark, kubebenchTargets string, timeout time.Duration) (*kubebench.OverallControls, error) {
```

Comment:
may be good to consolidate all args into a single structure called `kubebenchParams`.

Reply:
Thanks @JimBugwadia but where do you suggest should we create this struct? Here or create a new file to define it? Or inside the main.go?

Reply:
Hi @JimBugwadia , I have tried to consolidate all args into a single struct, please find some time to review, thanks!